app/javascript/components/wikilink.html

```diff
@@ -0,0 +1,6 @@
+<span>
+  <a v-if="id" :href="'/wiki/' + id">
+    <slot name="default"></slot>
```

Comment:
This might be a silly question, but are the `name="default"` attributes in this file doing anything? As I understand [the documentation here](https://vuejs.org/v2/guide/components.html#Named-Slots) named slots are only useful if the parent has some markup with an attribute `slot="header"`. I might be completely misunderstanding, though.

Reply:
You're correct `name="default"` isn't required. Will remove